src/avatar_action.cpp

```diff
@@ -860,6 +860,11 @@ void avatar_action::eat( avatar &you, const item_location &loc,
 void avatar_action::plthrow( avatar &you, item_location loc,
                              const cata::optional<tripoint> &blind_throw_from_pos )
 {
+    const auto ret = you.can_wield( *loc );
```

Comment:
Almost Always Avoid Auto
```suggestion
    const ret_val<bool> ret = you.can_wield( *loc );
```